portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,151 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Katie's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Questrial&display=swap" rel="stylesheet">
+    <style>
+      html {
+          scroll-behavior: smooth;
+      }
+    </style>
```

Comment:
Please place all styling in a CSS file (this called [separation of concerns](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns) in the style guide). Here and below.

The idea is that the html file should contain the structure and content, while the css file should contain the styling. With this set-up, it's possible to display the same content in many different ways, just by changing the css file. The following site is a great example: http://www.csszengarden.com/